packages/test/test-end-to-end-tests/src/test/bindToContextTests.spec.ts

```diff
@@ -1,192 +0,0 @@
-/*!
- * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
- * Licensed under the MIT License.
- */
-
-import { strict as assert } from "assert";
-import { IFluidHandle, IRequest } from "@fluidframework/core-interfaces";
-import { ITestObjectProvider } from "@fluidframework/test-utils";
-import {
-	describeFullCompat,
-	ITestDataObject,
-	TestDataObjectType,
-} from "@fluid-internal/test-version-utils";
-import { requestFluidObject } from "@fluidframework/runtime-utils";
-import { IContainerRuntimeBase } from "@fluidframework/runtime-definitions";
-import { IDataStoreWithBindToContext_Deprecated } from "@fluidframework/container-runtime-definitions";
-
-describeFullCompat("bindToContext tests", (getTestObjectProvider, apis) => {
-	const {
-		dataRuntime: { DataObject, DataObjectFactory },
-		containerRuntime: { ContainerRuntimeFactoryWithDefaultDataStore },
-	} = apis;
-	class InnerDataObject extends DataObject implements ITestDataObject {
-		public get _root() {
-			return this.root;
-		}
-
-		public get _context() {
-			return this.context;
-		}
-
-		public get _runtime() {
-			return this.runtime;
-		}
-	}
-	const innerDataObjectFactory = new DataObjectFactory(
-		"InnerDataObject",
-		InnerDataObject,
-		[],
-		[],
-	);
-
-	class OuterDataObject extends DataObject implements ITestDataObject {
-		public get _root() {
-			return this.root;
-		}
-
-		public get _context() {
-			return this.context;
-		}
-
-		public get _runtime() {
-			return this.runtime;
-		}
-
-		private readonly innerDataStoreKey = "innerDataStore";
-
-		protected async initializingFirstTime(): Promise<void> {
-			const innerDataStoreRouter = await this._context.containerRuntime.createDataStore(
-				innerDataObjectFactory.type,
-			);
-			const innerDataStore = await requestFluidObject<ITestDataObject>(
-				innerDataStoreRouter,
-				"",
-			);
-
-			this.root.set(this.innerDataStoreKey, innerDataStore.handle);
-
-			// IMPORTANT: Without calling bindToContext, requesting this inner object deadlocks (handle.get is fine)
```

Comment:
This is still true.  Maybe we should keep "documentation" of this in a test by having a test that _doesn't_ call bindToContext (since it's gone) and has a 100ms timeout or something to demonstrate the hang

Reply:
Until request is totally gone.  Or, since it's already deprecated, we can just delete these tests and forget about it.